Subject: do not repeat erase
From: Sascha Hauer <s.hauer@pengutronix.de>

I cannot think of why anyone would repeat the last erase
command by hitting return. So disable it.

Signed-off-by: Sascha Hauer <s.hauer@pengutronix.de>

---
 common/cmd_flash.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: common/cmd_flash.c
===================================================================
--- common/cmd_flash.c.orig
+++ common/cmd_flash.c
@@ -690,7 +690,7 @@ U_BOOT_CMD(
 );
 
 U_BOOT_CMD(
-	erase,   3,   1,  do_flerase,
+	erase,   3,   0,  do_flerase,
 	"erase   - erase FLASH memory\n",
 	"start end\n"
 	"    - erase FLASH from addr 'start' to addr 'end'\n"
